Update doc-concept-template.md (#42682)
grafana/grafana: contribute/templates/doc-concept-template.md

# Concept

The title of the concept topic will generally be a noun or a gerund. Examples include _Templates_, _Templating_, _Dashboards_, and _Panels_.

A concepts is an information type that explains a concept, idea, or provides an overview or workflow. In the introduction, such as this first paragraph or two, explain what the reader can expect to read.

[Permissions overview](https://grafana.com/docs/grafana/latest/permissions/overview/) is an example of a concept topic.

## Idea

A concept explains _what_ and _why_, but not _how_. If you are a new user, you might look for concept information to learn about what Grafana is, why it might be useful to you, and what the general workflow is.

## Workflow

A sample Grafana workflow might look as follows:

1. Install Grafana. <link to a task for installing Grafana>
1. Set up a data source. <link to a data source concept, which links to a data source task>
1. Create panels. <link to a panel concept, which links to a task>
1. Create dashboards. <link to a panel concept, which links to a task>
1. Enter queries. <link to a query editor concept>
1. Add users. <link to a user-management concept, which links to a task>

> 1. Create playlists. <link to a playlist concept or task>

Author: ```suggestion
1. Create playlists. <link to a playlist concept or task>

Try and link a concept to related information, such as a _task_ or _reference_.
```

Author: I intend on adding metadata for related content; note to self, see https://www.regisphilibert.com/blog/2018/04/hugo-optmized-relashionships-with-related-content/.

Author: @robbymilo, does this metadata look correct to you?

# Concept

The title of the concept topic will generally be a noun or a gerund. Examples include _Templates_, _Templating_, _Dashboards_, and _Panels_.

> A concepts is an information type that explains a concept, idea, or provides an overview or workflow. In the introduction, such as this first paragraph or two, explain what the reader can expect to read.

Reviewer: ```suggestion
A concept is an information type that helps readers _ understand_ by explaining a concept or idea or providing an overview of workflow. 

In the first sentence, explain what the concept is succinctly for easier reading. In the next sentence (or two or three) explain the value or benefit that is provided.
```

[Permissions overview](https://grafana.com/docs/grafana/latest/permissions/overview/) is an example of a concept topic.

## Idea

> A concept explains _what_ and _why_, but not _how_. If you are a new user, you might look for concept information to learn about what Grafana is, why it might be useful to you, and what the general workflow is.

Reviewer: ```suggestion
A concept explains _what_ and _why_, but not _how_. If you are a new user, you might look for concept information to learn about what Grafana is, why it might be useful to you, and what the general workflow is.

A sample concept might look like this:

# Example: About Grafana panel libraries

A library panel is a reusable panel that you can use in any dashboard. When you make a change to a library panel, that change propagates to all instances of where the panel is used. Library panels streamline reuse of panels across multiple dashboards.

You can save a library panel in a folder alongside saved dashboards.
```

## Workflow

A sample Grafana workflow might look as follows:

1. Install Grafana. <link to a task for installing Grafana>
1. Set up a data source. <link to a data source concept, which links to a data source task>
1. Create panels. <link to a panel concept, which links to a task>
1. Create dashboards. <link to a panel concept, which links to a task>
1. Enter queries. <link to a query editor concept>
1. Add users. <link to a user-management concept, which links to a task>
1. Create playlists. <link to a playlist concept or task>

> Try and link a concept to related information, such as a _task_ or _reference_.

Reviewer: I'd suggest remove the "Next steps" section as a manual update. I'd like to evaluate auto-generating this.
